ios/README.md

```diff
@@ -0,0 +1,72 @@
+
+## Prerequisites
+- iOS device (iOS version 11.0 or higher)
+- EMOTIV App for iOS
+- Emotiv headset (if you don't have, you can purchase one [here](https://www.emotiv.com))
+- Emotiv account (if you don't have, you can register [here](https://www.emotiv.com))
+- The client id/secret of your cortex app (if you don't have, you can register [here](https://www.emotiv.com/my-account/cortex-apps/))
+
```

Comment:
the example has a property for 'license', please add a guide for getting a license if the user needs it.

Reply:
ok, good suggestion.